bin/mochi.js

```diff
@@ -48,71 +48,105 @@ const blacklist = [
   "Tab added and finished loading"
 ];
 
-let mode = "starting";
-
 function sanitizeLine(line) {
   return line.trim().replace(/\\"/g, '"').replace(/\\"/g, '"');
 }
 
-function onGecko(line) {
+function onFrame(line, data) {
```

Comment:
true... I'll move this file to a place where prettier works and then we'll find other terrible things.